frameworks/Python/API-Hour/benchmark_config

```diff
@@ -20,8 +20,62 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "API-Hour",
-      "notes": "Python 3.4 + AsyncIO"
+      "display_name": "API-Hour+aiohttp.web+pgsql",
+      "notes": "Python 3 + AsyncIO + aiohttp.web + PostgreSQL"
+    }
```

Comment:
Needs a `,`. I just checked out your branch and there are errors because of this when attempting to run these tests.